lib/msal-core/src/telemetry/HttpEvent.ts

```diff
@@ -33,7 +33,7 @@ export default class HttpEvent extends TelemetryEvent {
     }
 
     public set queryParams(queryParams: any) {
-        this.event[EVENT_KEYS.QUERY_PARAMETERS] = Utils.generateQueryParametersString(queryParams);
+        // this.event[EVENT_KEYS.QUERY_PARAMETERS] = Utils.generateQueryParametersString(queryParams);
```

Comment:
Can you add a comment about why this is commented out (I assume this is temporary)?

Reply:
The refactor changed the visibility of a function in the telemetry work. I wanted @DarylThayil to take a look at this.

Reply:
why can't that function be public

Reply:
It is a member of serverRequestParameters class now. I'd prefer to keep it private so we don't start using the functionality outside. Is there another way to represent this event?

Reply:
I've changed generateQueryParameters to a static function, can we get this merged?